src/content/docs/en/guides/client-side-scripts.mdx

```diff
@@ -62,7 +62,18 @@ By default, `<script>` tags are processed by Astro.
 </script>
 ```
 
-To avoid bundling the script, you can use the `is:inline` directive.
+#### All processed scripts become module scripts
+
+This has several benefits:
+- Rendering is not blocked. The browser continues to process the rest of the html while the module script and its dependencies load.
+- The browser waits for html to be processed before executing module scripts. You do not need to listen for the "load" event.
+- `async` and `defer` attributes become unnecessary. Module scripts are always deferred.
```

Comment:
```suggestion
- `async` and `defer` attributes are unnecessary. Module scripts are always deferred.
```
I think this fits the same verb tense as the others. We're describing what *does* happen.